### ISSUE TITLE
No view found for id 0xe (unknown) for fragment FragmentContainer

### ISSUE THREAD

**Issue by melicaster** [xamarin/Xamarin.Forms#4973]
### Description
I have a Tabbed Page contained a Tabbed Page and 4 Content Page.

```
<TabbedPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="Admin">
  <!--Pages can be added as references or inline-->
    <TabbedPage Title="Tab">
        <ContentPage Title="tabbed content 1"/>
        <ContentPage Title="tabbed content 2"/>
    </TabbedPage>
    <ContentPage Title="tab2" />
    <ContentPage Title="tab3" />
    <ContentPage Title="tab4" />
    <ContentPage Title="tab5">
        <Label HorizontalOptions="CenterAndExpand" VerticalOptions="CenterAndExpand" Text="hello"/>
    </ContentPage>
</TabbedPage>
```

### Steps to Reproduce

1. Create a tabbed page
2. Put a new tabbed page into the current tabbed page.
3. Navigate between tabbed page and content page is OK. Once you click the last content page and want to navigate back to either 1 to 4 tabs, it will hit the error.

### Expected Behavior
Expected to navigate freely between tabbed page and content page.
### Actual Behavior
No view found for id 0xe (unknown) for fragment FragmentContainer
### Basic Information

- Version with issue:
- Last known good version: N/A
- IDE: Visual Studio 2017
- Platform Target Frameworks: 
  - iOS:  N/A
  - Android: 7.1
  - UWP:  N/A
- Android Support Library Version: 
- Nuget Packages: N/A
- Affected Devices: N/A

### Screenshots
![screenshot_1547321904](https://user-images.githubusercontent.com/15340082/51077678-c8479600-16e4-11e9-8792-f9ef1a0bb811.png)
![screenshot_1547321912](https://user-images.githubusercontent.com/15340082/51077680-cbdb1d00-16e4-11e9-9769-6305a350e0b0.png)

**Comment by kvpt** [comment]
These kind of errors are always related to dispose problem somewhere.
And it's that type of errors I try to fix at this moment for one of my app.
So as I had everything in place to test I give it a try.

At first, I thought it would be simple because I quickly noticed that [this line](https://github.com/xamarin/Xamarin.Forms/blob/411d2f60b228b82357e12619abdf730250d17146/Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs#L160) was wrong, because it's a list so  you have to use remove or clear but not add null.
But after that fix, the problem was the same, so I made a quick search and found [that post](https://stackoverflow.com/a/42310875) which suggest that the fragments was not clear and indeed it was the case.
So I added the missing part in the dispose of the fragmentPagerAdapter, it fix the issue and perhaps a memory leak too.

**Comment by melicaster** [comment]
where are we on this? 🙂

**Comment by Happypig375** [comment]
I hit this issue just now... Why are so many Xamarin.Forms bugs delayed from one release to another?

**Comment by melicaster** [comment]
where are we on this? I thought kvpt already provided fixes?

**Comment by samhouts** [comment]
closed by #4974
